libfwupdplugin/fu-crc.c

```diff
@@ -37,6 +38,7 @@ fu_crc8_full(const guint8 *buf, gsize bufsz, guint8 crc_init, guint8 polynomial)
 	}
 	return ~((guint8)(crc >> 8));
 }
+#endif
```

Comment:
Couldn't you just have this one `ifndef` wrap all 3?  Or is the concern the documentation?

Reply:
I was worried about the docs. I'll play -- I didn't like the idea of duplicating them in two source files.